tests/by-util/test_sort.rs

```diff
@@ -3,14 +3,24 @@ use crate::common::util::*;
 fn test_helper(file_name: &str, args: &str) {
     new_ucmd!()
         .arg(format!("{}.txt", file_name))
-        .args(&args.split(' ').collect::<Vec<&str>>())
+        .args(
+            &args
+                .split(' ')
+                .filter(|arg| !arg.is_empty())
```

Comment:
This looks like it could be simplified with [split_whitespace](https://doc.rust-lang.org/std/primitive.str.html#method.split_whitespace)

Reply:
Thanks, changed.